README.md

```diff
@@ -1,17 +1,106 @@
-# Griptape Tool Template
+# Griptape Plugins Template
 
-A Github template repository for creating Griptape tools.
+A Github template repository for creating Griptape plugins.
```

Comment:
Suggest "extensions" as it implies extending functionality and capabilities. Opinion, tho.

Reply:
I think I like extension more. Plugin, to me, suggests some sort of modification of core behavior. This template is for providing new, extended functionality.